browser/src/layer/tile/CanvasTileLayer.js

```diff
@@ -5555,11 +5574,8 @@ L.CanvasTileLayer = L.Layer.extend({
 							       ' at pos ' + destCol + ', ' + destRow + ' into delta at byte: ' + offset);
 				i += 4;
 				span *= 4;
-				// copy so this is suitably aligned for a Uint32Array view
-				this._unpremult.unpremultiply(delta, span, i);
-				// imgData.data[offset + 1] = 256; // debug - greener start
 				for (var j = 0; j < span; ++j)
-					imgData.data[offset++] = this._unpremult._bytes[j];
+					imgData.data[offset++] = delta[i+j];
```

Comment:
Now the data is always unpremultiplied before this point. At least, that's the intention.